appservice/src/createAppService/createAppService.ts

```diff
@@ -93,13 +93,13 @@ export async function createAppService(
     executeSteps.push(new SiteCreateStep(createOptions.functionAppSettings));
     const wizard: AzureWizard<IAppServiceWizardContext> = new AzureWizard(promptSteps, executeSteps, wizardContext);
 
-    // Ideally actionContext should always be defined, but there's a bug with the NodePicker. Create a 'fake' actionContext until that bug is fixed
+    // Ideally actionContext should always be defined, but there's a bug with the TreeItemPicker. Create a 'fake' actionContext until that bug is fixed
```

Comment:
has this not been fixed?

Reply:
Nope. https://github.com/Microsoft/vscode-azuretools/issues/120